### PR TITLE
Better image testing docs

### PR DESCRIPTION
**Pull request by etpinard**
@perigee @alexcjohnson @cldougl @mdtusz 

Better dockers docs, including info about `docker-machine` for Mac users.

### CODE REVIEW

File: test/image/README.md [hunk 1]
```diff
@@ -1,24 +1,46 @@
...
+Boot up docker machine (named `default`):
+
+```bash
+docker-machine start default
+```
+
+Set up the docker environment for `docker-compose`:
+
+```bash
+eval $(docker-machine env default)
```
Comment (etpinard) [inline test/image/README.md]: @alexcjohnson this should do the trick. 

Reference: https://github.com/docker/compose/issues/2180#issuecomment-147769429

**Comment by mdtusz** [comment]
Looks ok to me. 

Fun little oneliner for when you've got a lot of docker containers still that you want to remove:

`docker ps -a | xargs docker rm -f`

:dancer:

**Comment by cldougl** [comment]
:+1:  from me too! :whale: :whale: :whale:

File: test/image/README.md (hunk 1 above)
Comment (perigee) [inline test/image/README.md]: Seems my Mac don't need such work around :-( my `docker-compose` version `1.4.2`

Comment (etpinard) [inline test/image/README.md]: @alexcjohnson @cldougl what version of `docker-compose` are you using?

Comment (cldougl) [inline test/image/README.md]: docker-compose version: 1.5.1

Comment (alexcjohnson) [inline test/image/README.md]: yup, the eval did it for me. (I used it with quotes, as mentioned within the output of `docker-machine env default` itself)
Image tests work locally for me now! Well almost, some intermittent gl3d errors (`gl3d_marker-color.png` failed twice in a row, `gl3d_mirror-ticks.png` failed one of two times)

Does it save the images it makes somewhere when they fail?

Version: 1.5.1, just like @cldougl (installed today)

Comment (etpinard) [inline test/image/README.md]: > Does it save the images it makes somewhere when they fail?

Yes, look into `build/test_images` and `build/test_images_diff``.

Or even better, run `npm run start-image_viewer` to view the baseline, generated image, diff and mock side by side.

Comment (mdtusz) [inline test/image/README.md]: A good shell function to add to your rc file if you have lots of docker-machines (I have like 8 that I use depending) is [this](https://github.com/mdtusz/dotfiles/blob/master/.shellfn#L32).

**Comment by etpinard** [comment]
@perigee I'm going to merge this knowing that the updated instructions are working on @cldougl and @alexcjohnson Macs.

**Pull request merged**